src/components/IndicatorCard/RangeCard/RangeCard.scss

```diff
@@ -0,0 +1,112 @@
+@import '~style/variables';
+
+.wrapper {
+  @include shadow;
+
+  display: flex;
+  flex-direction: column;
+  position: relative; // .indicatorTypeMark is absolute, relative to this
+  background: $color-white;
+  border: none;
+  border-radius: 4px;
+  overflow: hidden; // clip the footer with the border radius
+}
+
+.smallCard {
+  height: 112px;
+}
+
+.largeCard {
+  height: 168px;
+}
+
+.indicatorName {
+  flex: 1;
+  position: absolute;
+  right: 54px;
+  font-weight: 500;
+
+  .smallCard & {
+    left: 24px;
+    top: 3px;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  .largeCard & {
+    left: 29px;
+    top: 4px;
+    font-size: 16px;
+    line-height: 1.1;
+  }
+}
+
+.indicatorTypeMark {
+  position: absolute;
+
+  .smallCard & {
+    top: 2px;
+    left: 4px;
+  }
+
+  .largeCard & {
+    top: 4px;
+    left: 9px;
+  }
+}
```

Comment:
There is some duplication here between the `RangeCard` and the `NumberCard`, but a lot of stuff is _slightly_ different. A little bit of repetition is better than a lottle bit of complexity.

Reply:
You say a little, I say a lottle - what you need is a little bot.